R/utils-classes.R

```diff
@@ -389,7 +395,8 @@ collect.bcdc_promise <- function(x, ...){
     url <- cc$url
     full_url <- cli$url_fetch(query = query_list)
   } else {
-    message("This record requires pagination to complete the request.")
+    chunk <- getOption("bcdata.chunk_limit", default = 1000)
+    message(glue::glue("This record requires {ceiling(number_of_records/chunk)} paginated requests to complete."))
```

Comment:
```suggestion
    message(glue::glue("This object has {number_of_records} records and requires {ceiling(number_of_records/chunk)} paginated requests to complete."))
```

Reply:
Yup, pretty much what I meant - just one suggested addition to add the total number of records so it's a little less abstract.